onnxmltools/convert/coreml/convert.py

```diff
@@ -84,7 +84,20 @@ def convert(model, name=None, doc_string=''):
     for output in outputs:
         output.name = context.get_onnx_name(output.name)
 
-    mb = ModelBuilder(name, doc_string)
+    # Convert CoreML description, author and license
+    metadata = spec.description.metadata
+    metadata_props = None
+    if metadata:
+        metadata_props = []
+        if metadata.shortDescription:
+            if not doc_string:
+                doc_string = metadata.shortDescription
```

Comment:
It means sometimes we add shortDescription and sometime we don't. For consistency, can we do something like creating a StringStringEntryProto to store this value (key: 'short description from CoreML model', value: metadata.shortDescription')?

Reply:
It means `shortDescription` is always used unless the user explicitly overrides it when calling `convert`. `shortDescription` should be shown in tools the same way as `doc_string`, adding another description property seems arbitrary. A better approach would be to discourage passing `doc_string` to `convert` as the CoreML file should already contain the description.

Reply:
Sounds good. I am fine with this change now.

Reply:
Done.